modules/i3s/src/lib/parsers/parse-i3s-tile-content.js

```diff
@@ -36,22 +36,17 @@ export async function parseI3STileContent(arrayBuffer, tile, tileset, options) {
   tile.content.attributes = {};
 
   if (tile.textureUrl) {
-    const uvDebugTexture = options.i3s.uvDebugTexture;
-    if (uvDebugTexture) {
-      tile.content.texture = uvDebugTexture;
-    } else {
-      const url = getUrlWithToken(tile.textureUrl, options.token);
-      const loader = FORMAT_LOADER_MAP[tile.textureFormat] || ImageLoader;
-      tile.content.texture = await load(url, loader);
-      if (loader === CompressedTextureLoader) {
-        tile.content.texture = {
-          compressed: true,
-          mipmaps: false,
-          width: tile.content.texture[0].width,
-          height: tile.content.texture[0].height,
-          data: tile.content.texture
-        };
-      }
+    const url = getUrlWithToken(tile.textureUrl, options.token);
+    const loader = FORMAT_LOADER_MAP[tile.textureFormat] || ImageLoader;
+    tile.content.texture = await load(url, loader);
+    if (loader === CompressedTextureLoader) {
+      tile.content.texture = {
```

Comment:
I'd love to standardize the output of the Compressed texture loaders so we don't need to do this remapping, before we publish 3.0.

Basically add the data in the data field instead of in the texture field?